src/garage/envs/grid_world_env.py

```diff
@@ -39,7 +42,13 @@ class GridWorldEnv(gym.Env):
     | 'G' : goal
     """
 
-    def __init__(self, desc='4x4'):
+    def __init__(self, desc='4x4', max_episode_length=math.inf):
```

Comment:
```suggestion
    def __init__(self, desc='4x4', max_episode_length=None):
```

Reply:
`GridWorldEnv` doesn't have a default time limit. In this case we should probably keep math.inf? 

Or we would have to check `max_episode_length` and have different logics for initializing EnvSpec - otherwise EnvSpec will get `max_episode_length == None`

Reply:
Please use `None`

Reply:
Again, I think it should be `None` which is a more typical way in python of representing "no limit"